server/.env.example

```
@@ -26,6 +26,9 @@ GROQ_MODEL_AGENTS=llama-3.3-70b-versatile
 # Chatbot uses the fast 8B model to stay well under rate limits.
 GROQ_MODEL_CHAT=llama-3.1-8b-instant
 
+# ── River ML Anomaly Detection ───────────────────────────────────────────────
+RIVER_ANOMALY_THRESHOLD=0.75    # Anomaly score threshold (0.0-1.0)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)